src/board/m5stack/M5CORE2.h

```diff
@@ -0,0 +1,244 @@
+
+#endif /* ESP_PANEL_LCD_BUS_TYPE */
+
+/**
+ * LCD Venbdor Initialization Commands.
+ *
+ * Vendor specific initialization can be different between manufacturers, should consult the LCD supplier for
+ * initialization sequence code. Please uncomment and change the following macro definitions. Otherwise, the LCD driver
+ * will use the default initialization sequence code.
+ *
+ * There are two formats for the sequence code:
+ *   1. Raw data: {command, (uint8_t []){ data0, data1, ... }, data_size, delay_ms}
+ *   2. Formater: ESP_PANEL_LCD_CMD_WITH_8BIT_PARAM(delay_ms, command, { data0, data1, ... }) and
+ *                ESP_PANEL_LCD_CMD_WITH_NONE_PARAM(delay_ms, command)
+ */
+// #define ESP_PANEL_LCD_VENDOR_INIT_CMD                                          \
+//     {                                                                          \
+//         {0xFF, (uint8_t []){0x77, 0x01, 0x00, 0x00, 0x10}, 5, 0},              \
+//         {0xC0, (uint8_t []){0x3B, 0x00}, 2, 0},                                \
+//         {0xC1, (uint8_t []){0x0D, 0x02}, 2, 0},                                \
+//         {0x29, (uint8_t []){0x00}, 0, 120},                                    \
+//         or                                                                     \
+//         ESP_PANEL_LCD_CMD_WITH_8BIT_PARAM(0, 0xFF, {0x77, 0x01, 0x00, 0x00, 0x10}), \
+//         ESP_PANEL_LCD_CMD_WITH_8BIT_PARAM(0, 0xC0, {0x3B, 0x00}),                   \
+//         ESP_PANEL_LCD_CMD_WITH_8BIT_PARAM(0, 0xC1, {0x0D, 0x02}),                   \
+//         ESP_PANEL_LCD_CMD_WITH_NONE_PARAM(120, 0x29),                               \
+//     }
+
+/* LCD Color Settings */
+/* LCD color depth in bits */
+#define ESP_PANEL_LCD_COLOR_BITS    (16)        // 8/16/18/24
+/*
+ * LCD RGB Element Order. Choose one of the following:
+ *      - 0: RGB
+ *      - 1: BGR
+ */
+#define ESP_PANEL_LCD_BGR_ORDER     (1)         // 0/1
+#define ESP_PANEL_LCD_INEVRT_COLOR  (1)         // 0/1
+
+/* LCD Transformation Flags */
+#define ESP_PANEL_LCD_SWAP_XY       (0)         // 0/1
+#define ESP_PANEL_LCD_MIRROR_X      (0)         // 0/1
+#define ESP_PANEL_LCD_MIRROR_Y      (0)         // 0/1
+
+/* LCD Other Settings */
+/* IO num of RESET pin, set to -1 if not use */
+#define ESP_PANEL_LCD_IO_RST        (-1)
+#define ESP_PANEL_LCD_RST_LEVEL     (0)         // 0: low level, 1: high level
+
+#endif /* ESP_PANEL_USE_LCD */
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+//////////////////////////// Please update the following macros to configure the touch panel ///////////////////////////
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+/* Set to 1 when using an touch panel */
+#define ESP_PANEL_USE_TOUCH         (1)         // 0/1
+#if ESP_PANEL_USE_TOUCH
+/**
+ * Touch controller name.
+ */
+#define ESP_PANEL_TOUCH_NAME        FT5x06
```

Comment:
Here also add a comment to explain the compatibility of ft5x06 and ft6xxx.

Reply:
done